WordPress/Classes/ViewRelated/Activity/ActivityListViewModel.swift

```diff
@@ -168,6 +233,16 @@ class ActivityListViewModel: Observable {
         return firstYear != currentYear || secondYear != currentYear
     }
 
+    private func applyingNewFilter(after: Date? = nil, before: Date? = nil, group: [ActivityGroup]) -> Bool {
```

Comment:
Thoughts on prefixing with `is` to make it a bit clearer?

```suggestion
    private func isApplyingNewFilter(after: Date? = nil, before: Date? = nil, group: [ActivityGroup]) -> Bool {
```

Reply:
Done in 6f4fb96128a48d3948cf96d3b5fc1cdb91a8d36a